translation/en/RedirectedResources/assets/abdata/adv/scenario/c30/20/1100/translation.txt

```diff
@@ -10,15 +10,15 @@
 今日、俺は叶えることができた。=Today, I'll be able to make that become reality.
 （絶対に……[H名]を幸せにしてみせる……！）=(No matter what... I'll definitely make [H名] happy...!)
 「それではこれより、式を執り行います」=“Then without further ado, let's start the ceremony.”
-「新たなご出発の証として、お二人には誓いの言葉を立てていただきます」=“As the proof of a fresh departure, you both have to make certain vows.”
+「新たなご出発の証として、お二人には誓いの言葉を立てていただきます」=“As proof of your new beginnings, we ask of you to share your vows.”
 牧師=Pastor
-「[P名]さん、あなたは今、この女性を妻とし夫婦になろうとしています」=“Mr. [P名], you are now about to take this woman to be your wife.”
-「汝、健やかなるときも、病めるときも、喜びのときも、悲しみのときも、富めるときも、貧しいときも」=“Do you promise, in times of sickness or in health, in times of happiness or sadness, whether you are rich or poor,”
-「これを愛し、これを敬い、これを慰め、これを助け、その命ある限り、真心を尽くすことを誓いますか？」=“to love her, to respect her, to comfort her, to support her, till the end of days, to the full extent of your abilities?”
-「はい、誓います」=“Yes, I vow.”
-「[H名]さん、あなたは今、この男性を夫とし夫婦になろうとしています」=“Miss [H名], you are now about to take this man as your husband.”
-「では、二人の誓いの印として、指輪の交換を致します」=“Then, as the proof of your vows, you may now exchange the rings.”
-「初めに新郎から新婦へ」=“First from the groom to the bride.”
+「[P名]さん、あなたは今、この女性を妻とし夫婦になろうとしています」=“[P名]-san, you are now about to take this woman to be your wife.”
+「汝、健やかなるときも、病めるときも、喜びのときも、悲しみのときも、富めるときも、貧しいときも」=“In times when you are in good health, when you are ill, when you are happy, when you are sad, when you are rich, and when you are poor.”
+「これを愛し、これを敬い、これを慰め、これを助け、その命ある限り、真心を尽くすことを誓いますか？」=“You will love them, respect them, comfort them, help them. Do you vow to do your very best for as long as you can?”
+「はい、誓います」=“Yes, I promise.”
+「[H名]さん、あなたは今、この男性を夫とし夫婦になろうとしています」=“[H名]-san, you are now about to take this man as your husband.”
```

Comment:
I think it's fine to have small differences like this, especially since they are already done. It adds some variation while keeping the meaning the same.